src/app/helptext/topbar.ts

```diff
@@ -93,6 +95,18 @@ Login or signup is required.'),
         title: T('Verify Email Address'),
         message: T('A message with verification instructions has been sent to the new email \
 address. Please verify the email address before continuing.')
+    },
+    changePasswordDialog: {
+        pw_form_title_name: T('Change Administrator Password'),
+        pw_current_pw_placeholder : T('Current Password'),
+        pw_new_pw_placeholder : T('New Password'),
+        pw_new_pw_tooltip : T('Passwords cannot contain a <b>?</b>. Passwords should\
+         be at least eight characters and contain a mix of lower and\
```

Comment:
there should only be one space at the beginning of this text on each line, the translation extractor will pull out multiple spaces in this case